atc/exec/task_step.go

```diff
@@ -291,8 +273,8 @@ func (step *TaskStep) imageSpec(logger lager.Logger, repository *artifact.Reposi
 	return imageSpec, nil
 }
 
-func (step *TaskStep) containerInputs(logger lager.Logger, repository *artifact.Repository, config atc.TaskConfig, metadata db.ContainerMetadata) ([]worker.InputSource, error) {
-	inputs := []worker.InputSource{}
+func (step *TaskStep) containerInputs(logger lager.Logger, repository *build.Repository, config atc.TaskConfig, metadata db.ContainerMetadata) ([]worker.FooBarInput, error) {
```

Comment:
Why we do have `FooBarInput` here?  Should we have a more meaningful name?

Reply:
yup, its our latest TODO. Will update the pr with changes for this.